ti_mocha_tests/ti.filesystem.test.js

```diff
@@ -431,6 +431,23 @@ describe("filesystem", function() {
 		}
 		finish();
 	});
+	//TIMOB-20440
+	it(“applicationDataDirectory”, function(finish) {
```

Comment:
[ERROR] Unexpected character '“' [line 435, column 4]

I think you used the wrong double inverted commas here. Wrong character.